torch/onnx/_onnx_supported_ops.py

```diff
@@ -2,9 +2,10 @@
 from typing import Dict, List, Union
```

Comment:
I am looking to deprecate this file and refactor part of its functionality into the doc gen script and the new registry. So I am not moving it this time.